zai_sapi/testing/catch2.hpp

```diff
@@ -112,19 +176,26 @@ extern "C" {
         ZAI_SAPI_TSRMLS_FETCH();                                         \
         __ZAI_SAPI_TEST_CASE_BEGIN()                                     \
         if (__ZAI_SAPI_TEST_CASE_STUB) {                                 \
-            bool zai_sapi_test_case_stub_included = true;                \
-            zend_try {                                                   \
+            volatile bool                                                \
+                zai_sapi_test_case_stub_included = true;                 \
+            zend_first_try {                                             \
                 zai_sapi_test_case_stub_included =                       \
                     zai_sapi_execute_script(                             \
                         __ZAI_SAPI_TEST_CASE_STUB);                      \
             } zend_catch {                                               \
                 zai_sapi_test_case_stub_included = false;                \
             } zend_end_try();                                            \
             REQUIRE(zai_sapi_test_case_stub_included);                   \
+            if (!zai_sapi_test_case_stub_included) {                     \
```

Comment:
REQUIRE() aborts the program altogether, i.e. the goto in the branch will never be executed.

Reply:
Specifically, it doesn't abort the program, it returns from the test case method/function ...